cmd/eval_test.go

```diff
@@ -676,6 +676,38 @@ func removeBuiltin(builtins []*ast.Builtin, name string) []*ast.Builtin {
 	return cpy
 }
 
+// Nearly identical to TestEvalWithOptimizeBundleData, but uses
+// Rego entrypoint annotations instead of explicitly providing
+// the entrypoints as CLI arguments.
+func TestEvalWithRegoEntrypointAnnotations(t *testing.T) {
+	files := map[string]string{
+		"test.rego": `
+package test
+default p = false
+# METADATA
+# entrypoint: true
+p { q }
+q { input.x = data.foo }`,
+		"data.json": `
+			{"foo": 1}`,
+	}
+
+	test.WithTempFS(files, func(path string) {
+		params := newEvalCommandParams()
+		params.bundlePaths = repeatedStringFlag{
```

Comment:
Do we not have [a helper ](https://github.com/philipaconrad/opa/blob/7faaab38fe70960cdc6cb0c958d339dbb272ba1a/cmd/eval.go#L706) for this? I think we can use `params.bundlePaths.Set(path)` here.

Reply:
I was entirely unaware that there was a helper for that, but I'll check it out. Thank you for linking it!

Reply:
Rolled in a refactoring to use the helper methods consistently in that test file. :+1: